sqldelight-studio-plugin/testData/formatting/Formatting_after.sq

```diff
@@ -0,0 +1,33 @@
+import java.util.List;
+import java.util.ArrayList;
+
+CREATE TABLE test (
+    _id INTEGER NOT NULL PRIMARY KEY,
+    abs_list BLOB AS List NOT NULL,
+    array_list BLOB AS ArrayList
+);
+
+some_select:
+SELECT * FROM test;
+
+/** This is
```

Comment:
alright so this is actually way harder than i originally though, since you're not just changing spacing anymore you're adding a new character (the `*` at the beginning of the new line). The java formatter has this class `InjectedLanguageBlockBuilder` to handle it since you have to edit the AST tree but i dont want to get into that yet